### ISSUE TITLE
PCIe x16 version

### ISSUE THREAD

**Issue by renne** [KrutavShah/PCIE-to-MXM-Adapter#6]
This is a great project! :+1: 

PCIe x1 is quite a bottleneck for modern GPUs.
I suggest to upgrade the PCB to PCIe x16 like the [EGX-MXM-AD](https://www.adlinktech.com/Products/Embedded_Graphics/embedded-graphics-solutions/EGX-MXM-AD?lang=en) or the [Eurocom](https://eurocom.com/ec/multigpu()multiGPU) riser cards.

**Comment by KrutavShah** [comment]
I agree, x1 is definitely a bottleneck from my testing. I do have an x16 design that is missing a few components here and there, so not complete, but I have not released that here because I feel like the interest in this project has fallen away after a while of not being updated.

If this x16 version (with 4x DisplayPorts/3x DP + 1x HDMI) is something you still want to see, I would be inclined against adding all the design files here, simply because the ones for x1 design already available here have been copied by others without credit. I feel like it may not be worth bothering with as a result. I am, however, open to considering this again in the future if there is enough interest. Here is what the design currently looks like:
<img width="644" alt="image" src="https://user-images.githubusercontent.com/30164598/222871671-7b66d977-1242-444b-a8a4-f2e54129b2f9.png">

**Comment by renne** [comment]
Great! :+1: 
Please make sure the positions of the video ports match an easily available bezel.
It would also be great to have an universal PCB layout which allows to solder HDMI or Displayport sockets at will.

**Comment by KrutavShah** [comment]
The PCB layout matches Nvidia's 10/20 series PCI brackets, so you could get one of those and the ports should line up exactly. On the front page of this repository, I have one of these GTX 1060 brackets installed on my prototype which you take a look at if you'd like. Quadro brackets should also work. I do not think AMD brackets line up with the ports, though.

**Comment by RansomScientist** [comment]
> I agree, x1 is definitely a bottleneck from my testing. I do have an x16 design that is missing a few components here and there, so not complete, but I have not released that here because I feel like the interest in this project has fallen away after a while of not being updated.
> 
> If this x16 version (with 4x DisplayPorts/3x DP + 1x HDMI) is something you still want to see, I would be inclined against adding all the design files here, simply because the ones for x1 design already available here have been copied by others without credit. I feel like it may not be worth bothering with as a result. I am, however, open to considering this again in the future if there is enough interest. Here is what the design currently looks like: <img alt="image" width="644" src="https://user-images.githubusercontent.com/30164598/222871671-7b66d977-1242-444b-a8a4-f2e54129b2f9.png">

Hey, thank you very much for your design, what intrigues me though is if you have considered trace pair impedance and length matching? With 1x it might not be too much of a problem but with 16x I'm guessing it would be extremely important but I can't see any of that in the image?

**Comment by KrutavShah** [comment]
> Hey, thank you very much for your design, what intrigues me though is if you have considered trace pair impedance and length matching? With 1x it might not be too much of a problem but with 16x I'm guessing it would be extremely important but I can't see any of that in the image?

Yes, the impedance of the differential pairs has been set in the PCB editor to meet a target of around 85 ohms, the standard for PCIe gen 3. As for the length matching, individual traces within pairs will be matched, but the inter-pair skew has some room for variation, which is why the separate lanes themselves are not matched. This shouldn't be much of an issue here. The goal here is to keep the lengths of the pairs short to minimize unnecessary losses.

This is from a Texas Instruments PCIe Layout Guidelines document:
<img width="315" alt="image" src="https://user-images.githubusercontent.com/30164598/224573351-0baed9ed-47ac-471a-aba1-8b387739303a.png">

**Comment by RansomScientist** [comment]
> > Hey, thank you very much for your design, what intrigues me though is if you have considered trace pair impedance and length matching? With 1x it might not be too much of a problem but with 16x I'm guessing it would be extremely important but I can't see any of that in the image?
> 
> Yes, the impedance of the differential pairs has been set in the PCB editor to meet a target of around 85 ohms, the standard for PCIe gen 3. As for the length matching, individual traces within pairs will be matched, but the inter-pair skew has some room for variation, which is why the separate lanes themselves are not matched. This shouldn't be much of an issue here. The goal here is to keep the lengths of the pairs short to minimize unnecessary losses.
> 
> This is from a Texas Instruments PCIe Layout Guidelines document: <img alt="image" width="315" src="https://user-images.githubusercontent.com/30164598/224573351-0baed9ed-47ac-471a-aba1-8b387739303a.png">

Good to know, if the design ever gets published or you feel comfortable sharing it in private it would be really nice if you could let me know. I would also happily share tests results if I build one.
